<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SliderSettings</class>
 <widget class="QWidget" name="SliderSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="2" column="1">
    <widget class="QComboBox" name="orientationComboBox">
     <item>
      <property name="text">
       <string>Horizontal</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Vertical</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="disabledCheckBox">
     <property name="text">
      <string>Disabled</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="valueLineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="orientationLabel">
     <property name="text">
      <string>Orientation</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="invertedCheckBox">
     <property name="text">
      <string>Inverted</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="valueLabel">
     <property name="text">
      <string>Value</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="MaterialSlider" name="slider">
     <property name="maximum">
      <number>100</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialSlider</class>
   <extends>QAbstractSlider</extends>
   <header location="global">QtMaterialWidgets/slider.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
